syntax = "proto3";

service HeatingService{
    rpc getSystemTempSetting(Empty) returns(IntResponse);
    rpc changeSystemTempSettings(IntRequest) returns(StringResponse);
    rpc toggleHeatingSystemStatus(Empty) returns(StringResponse);
}

message StringResponse{
    string text = 1;
}

message IntResponse{
    int32 value = 1;
}

message IntRequest{
    int32 value = 1;
}

message Empty{}